WIP - improve synopsises (#931)
cosmos/sdk-tutorials: academy/2-main-concepts/events.md

<HighlightBox type="synopsis">

> Dedicate some time to events in the Cosmos SDK - what they are, how they are useful, and how applications should implement them.
> 
> You should understand the concepts covered in the following sections before diving into events:

Reviewer: ```suggestion
Dedicate some time to events in the Cosmos SDK: learn what events are, how events are useful, and how events are implemented in applications.

Before diving into events, start with understanding the concepts covered in the following sections:
```

* [Transactions](./transactions.md)
* [Messages](./messages.md)
* [Modules](./modules.md)
* [Protobuf](./protobuf.md)

> You can find some code examples at the end of this section, in which events are implemented in the checkers blockchain.

Reviewer: ```suggestion
Code examples are provided at the end of this section. Examples show events implemented in the checkers blockchain.
```
